ckan/common.py

```diff
@@ -6,12 +6,107 @@
 # NOTE:  This file is specificaly created for
 # from ckan.common import x, y, z to be allowed
 
+import flask
+import pylons
 
-from pylons.i18n import _, ungettext
-from pylons import g, c, request, session, response
+from flask.ext.babel import gettext as flask_gettext
```

Comment:
Importing in the style of `flask.ext.babel` is deprecated in flask 0.11[1] in favor of `flask_babel`. Maybe we should use the new format directly so that we are not a bad example and do not have problems later?! 
[1] https://github.com/pallets/flask/issues/1135